cmd/ebrelayer/replay.go

```diff
@@ -78,94 +79,164 @@ func RunReplayEthereumCmd(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-// RunReplayCosmosCmd executes initRelayerCmd
-func RunReplayCosmosCmd(cmd *cobra.Command, args []string) error {
-	// Validate and parse arguments
-	if len(strings.Trim(args[0], "")) == 0 {
-		return errors.Errorf("invalid [tendermint-node]: %s", args[0])
-	}
-	tendermintNode := args[0]
+// RunReplayCosmosBurnLockCmd replay missed burn lock events from cosmos
+func RunReplayCosmosBurnLockCmd(cmd *cobra.Command, args []string) error {
```

Comment:
Let's use cobra and flags instead of grabbing positional arguments from the CLI.